### DISCUSSION TITLE
大佬能加个识别vmware的unity窗口的功能嘛

### DISCUSSION THREAD

**Discussion by DaTouJun** [Planshit/Tai#130] [Ideas]
现在如果使用虚拟机都统一识别为了vmware workstation
使用unity模式是能在任务栏当正常应用一起用的
（提一下，可能用的人不是特别多

**Comment by noberumotto** [comment]
没用过unity窗口，得研究一下

**Comment by noberumotto** [comment]
非常抱歉，这个功能没办法加入。unity模式还是属于在虚拟机中使用的应用，不应该与客户机中的数据合并。
